ts_src/types.ts

```diff
@@ -77,7 +77,22 @@ export interface Tapleaf {
   version?: number;
 }
 
-export type Taptree = Array<[Tapleaf, Tapleaf] | Tapleaf>;
+export const TAPLEAF_VERSION_MASK = 0xfe;
```

Comment:
I'm not sure if the place for this constant is in `types`.
Maybe it will be extracted to a BIP341 module (see https://github.com/bitcoinjs/bitcoinjs-lib/issues/1780) together with `isTapleaf`, `isTaptree` in the future. OK to be here for now.

Reply:
Yeah, I really didn't have a good place for it.